modules/microshift-adding-containers-to-blueprint.adoc

```diff
@@ -51,7 +51,7 @@ $ rpm2cpio microshift-release-info-${VERSION}.noarch.rpm | cpio -idmv
 +
 [source,terminal]
 ----
-$ jq -r '.images | .[] | ("[[containers]]\nsource = \"" + . + "\"\n")' ./usr/share/microshift/release/release-$(uname -i).json
+$ jq -r '.images | .[] | ("[[containers]]\nsource = \"" + . + "\"\n")' ./usr/share/microshift/release/release-$(uname -m).json
```

Comment:
I'm guessing this command doesn't require sudo but saw the other ones did so wanted to make sure.

Reply:
I'm not too sure but I can check, @ShaunaDiaz Do you know about this one either?

Reply:
That module is no more. But that would be a question for QE in any case.